axelarnetwork/axelar-core
# feat: update the gateway bytecode to allow prevOwner to sign commands

## Description
This PR updates the bytecode of AxelarGateway due to changes in https://github.com/axelarnetwork/ethereum-bridge/pull/16, which allows a previous owner to sign commands except for `ownershipTransfer`.

## Todos


## Steps to Test
Compile the contracts in ethereum-bridge, and check if the bytecode matches the one in this PR. Alternatively, you can run a local network, sign a token deployment and then an ownership transfer, submit the ownership transfer first and then see if the token deployment can still be submitted and executed. 

## Expected Behaviour

## Other Notes
